recipes-kernel/kernel-modules/kernel-module-qca6564_git.bb

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2019 Digi International Inc.
+
+SUMMARY = "Qualcomm QCA6564 wireless driver module"
+LICENSE = "ISC"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/ISC;md5=f3b90e78ea0cffb20bf5cca7947a896d"
+
+# Reference Qualcomm tag/version
+PV = "v4.2.89.63"
+
+SRCBRANCH = "qca6564/master"
+SRCREV = "2e780b29209ca6b0f71442be5fd36cf0ddddc661"
+
+SRC_URI = "\
+    git://github.com/digi-embedded/qcacld-2.0.git;protocol=git;branch=${SRCBRANCH} \
+    file://modprobe-qca6564.conf \
+"
+
+LOCALVERSION ?= "-${SRCBRANCH}"
```

Comment:
Please use:

PV = "...+${SRCPV}" 

so we keep track of the revision. The localversion seems useless here as this is a module.

Reply:
I'll add ${SRCPV} to the PV variable above, thanks.